src/journal/sd-journal.c

```diff
@@ -1337,7 +1337,7 @@ static int add_any_file(
                 goto finish;
         }
 
-        r = journal_file_open(fd, path, O_RDONLY, 0, false, false, NULL, j->mmap, NULL, NULL, &f);
+        r = journal_file_open(fd, path, O_RDONLY, 0, false, 0, false, NULL, j->mmap, NULL, NULL, &f);
```

Comment:
Instead of `0`, should we use `-1` here? If `0` is passed, then `MIN_COMPRESS_THRESHOLD` is used. This is different from the original behavior. Is it really what you want?

Reply:
In this case, it doesn't matter, because compress is hardcoded to false.

Reply:
Indeed.